wagtail/bin/wagtail.py

```diff
@@ -242,9 +242,20 @@ def _rewrite_file(self, filename):
         return change_count
 
 
+class Version(Command):
+    description = "List which version of watch you are using"
+
+    def run(self):
+        import wagtail
+        version = wagtail.get_version(wagtail.VERSION)
+
+        print("You are using Wagtail %(version)s" % {'version': version})
+
+
 COMMANDS = {
     'start': CreateProject(),
     'updatemodulepaths': UpdateModulePaths(),
+    '--version': Version()
```

Comment:
I think this should be simply 'version'; the '--' suggests it's an argument for other commands, rather than a command itself.

Reply:
I respectfully disagree with you, @tomdyson. I think using the double-hyphen for the version output command is very common. For example, big names that are a part of our stack use it: `python --version`, `node --version`, `npm --version`. Using the double-hyphen style would maintain consistency with those.

Reply:
You've convinced me @Scotchester! I withdraw my comment, @KalobTaulien :)

Reply:
Heh that's why I went with --version because Python does it. I'll revert shortly. Thanks both of you!

Reply:
I still wouldn't call this it's own "sub command" though. If you ever run `--help` on a tool that uses subcommands, `--version` is always listed as an option and not a command.

`npm` does actually have a `version` command but this is different to `npm --version`